api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+		xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+		
+		
+    <changeSet author="aman (generated)" id="name_phonetics-1598359669281-94">
+        <createTable tableName="name_phonetics">
+            <column autoIncrement="true" name="name_phonetics_id" type="INT">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="rendered_string" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="person_name_id" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="field" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="renderer_class_name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="aman (generated)" id="name_phonetics-1598359669281-519">
```

Comment:
Did we have this in the original sqldiff?

Reply:
MySQL automatically creates an index for a foreign key. Thus this index got created in db and liquibase generated a change set. I found this here - https://dev.mysql.com/doc/refman/8.0/en/constraint-foreign-key.html

Reply:
Wouldn't this index also get automatically created by mysql after running the liquibase changeset which creates the foreign key?

Reply:
@dkayiwa , you are right. Its obvious. My bad. I was looking at whether PostgreSQL creates indexes for foreign keys and it appears it does not( See [here](https://stackoverflow.com/questions/970562/postgres-and-indexes-on-foreign-keys-and-primary-keys) ). So, I went looking for articles how much search time will be reduced if we set the index. I found this [article](https://www.cybertec-postgresql.com/en/index-your-foreign-key/) and it appears that on large databases we should have this index. If we are having it in MySQL then we should probably have it in PostgreSQL as well.

Its your call whether we should have or remove this index ?

Reply:
Fair enough! 😊

Reply:
So the next question, why are you using a name that is already used by the foreign key constraint?

Reply:
Actually, its not me doing it , its what liquibase generated :sweat_smile: . Similar situations are present all over the schema file. For eg consider constraint name : address_for_person [here](https://github.com/openmrs/openmrs-core/blob/288f9251ecce8997bce4ee92608a23c741f8e135/api/src/main/resources/org/openmrs/liquibase/snapshots/schema-only/liquibase-schema-only-2.3.x.xml#L2975) and [here](https://github.com/openmrs/openmrs-core/blob/288f9251ecce8997bce4ee92608a23c741f8e135/api/src/main/resources/org/openmrs/liquibase/snapshots/schema-only/liquibase-schema-only-2.3.x.xml#L4936)

Reply:
Oh i see. 😊